### PR TITLE
Add warning about AdBlocker

### PR DESCRIPTION
**Pull request by tengqm**
Closes: #6745

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6786)

### CODE REVIEW

File: docs/tutorials/kubernetes-basics/index.html [hunk 1]
```diff
@@ -26,6 +26,7 @@ <h2>Kubernetes Basics</h2>
         <li>Debug the containerized application</li>
         </ul>
         <p>The tutorials use Katacoda to run a virtual terminal in your web browser that runs Minikube, a small-scale local deployment of Kubernetes that can run anywhere. There's no need to install any software or configure anything; each interactive tutorial runs directly out of your web browser itself.</p>
+	<p>Note that you may have to disable or customize the AdBlocker (if any) in your browser to view the modules.</p>
```
Comment (Bradamant3) [inline docs/tutorials/kubernetes-basics/index.html]: Capitalizing "AdBlocker" might lead users to think that we're referring to a specific ad blocker. Let's keep the reference more generic, and maybe rephrase a little, to something like:
"You might need to disable or customize any ad blockers that you've added to your browser." (Don't need the parenthetical "if any" ...)
Thanks!

Comment (tengqm) [reply docs/tutorials/kubernetes-basics/index.html]: ok

**Comment by BenHall** [comment]
Thank you for letting us know. This was due to a JavaScript error as we expected our JS Tracker to load, but it was blocked via the adblocker.

This should now be resolved and the site work even with adblockers.

**Comment by tengqm** [comment]
Okay.
